src/main/webapp/WEB-INF/web.xml

```diff
@@ -199,6 +199,17 @@ http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
             <transport-guarantee>CONFIDENTIAL</transport-guarantee>
         </user-data-constraint>
     </security-constraint>
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>PartialHelpPages</web-resource-name>
+            <description>Partial help pages cannot be accessed</description>
+            <url-pattern>/help/*</url-pattern>
```

Comment:
I'd prefer the directory to be named `partials` instead of `help`; this way, it can be used to contain other page partials in future. Furthermore, the word `help` is already found in the individual filenames, and should not be repeated.